5.5/run-mysqld.sh

```diff
@@ -1,4 +1,9 @@
-#!/bin/bash -e
+#!/bin/bash
+
+# For SCL enablement
+. /var/lib/mysql/.bashrc
```

Comment:
$HOME/.bashrc ? or is this script run as root?

maybe I would suggest to add a new variable 'MYSQL_DATA_PATH'